src/objects/IPrioritize.java

```diff
@@ -0,0 +1,20 @@
+package objects;
+
+public interface IPrioritize {
+
+    String HIGH_LEVEL = "High level";
+
+    String MEDIUM_LEVEL = "Medium level";
+
+    String LOW_LEVEL = "Low level";
+
+    int VEHICLE_LEVEL_YEAR = 2010;
+
+    int VEHICLE_LEVEL_PRICE = 9999;
+
+    int HOME_LEVEL_PRICE = 59999;
+
+    int HOME_LEVEL_M2 = 30;
```

Comment:
in other classes you don't have empty lines between variables, lets make it the same everywhere
```suggestion
    String HIGH_LEVEL = "High level";
    String MEDIUM_LEVEL = "Medium level";
    String LOW_LEVEL = "Low level";
    int VEHICLE_LEVEL_YEAR = 2010;
    int VEHICLE_LEVEL_PRICE = 9999;
    int HOME_LEVEL_PRICE = 59999;
    int HOME_LEVEL_M2 = 30;
```